docs/customize/administration_interface.md

```diff
@@ -0,0 +1,398 @@
+# Administration panel views
+
+## Summary
+
+The following document is a developer guide to the invenio administration panel, detailing its programmatic interface and usage.
+
+## Intended audience
+
+This guide is intended for system administrators and developers of InvenioRDM.
+
+Overview
+
+Invenio administration panel gives the administrator a set of tools to effectively manage an instance of InvenioRDM. 
+
+## Domain dictionary
+
+The following terms are introduced to facilitate defining the problem
```

Comment:
```suggestion
The following terms are introduced to facilitate defining the problem:
```
Do we have a problem? :)

Reply:
@alejandromumo still in thesis writing mode ;)